test/integration/server.js

```diff
@@ -153,10 +153,12 @@ helpers.stubUtxos = function(server, wallet, amounts, cb) {
       return {
         txid: helpers.randomTXID(),
         vout: Math.floor(Math.random() * 10 + 1),
-        satoshis: helpers.toSatoshi(amount).toString(),
+        satoshis: helpers.toSatoshi(amount),
```

Comment:
That's because ``txp.getBitcoreTx();`` expects inputs amount in to be a Number. Insight seem to return numeric value: https://test-insight.bitpay.com/api/addr/mmKjy8rB6A97BoqewPxxbf7bwZWAnyzW21/utxo